backend/src/seeds/gmsUserList.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { entities } from '@entity/index'
+// import { createTestClient } from 'apollo-server-testing'
+
+import { CONFIG } from '@/config'
+import { createServer } from '@/server/createServer'
+import { backendLogger as logger } from '@/server/logger'
+
+CONFIG.EMAIL = false
+
+const context = {
+  token: '',
+  setHeaders: {
+    push: (value: { key: string; value: string }): void => {
+      context.token = value.value
+    },
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    forEach: (): void => {},
+  },
+  clientTimezoneOffset: 0,
+}
+
+export const cleanDB = async () => {
+  // this only works as long we do not have foreign key constraints
+  for (const entity of entities) {
+    await resetEntity(entity)
+  }
+}
+
+const resetEntity = async (entity: any) => {
+  const items = await entity.find({ withDeleted: true })
+  if (items.length > 0) {
+    const ids = items.map((e: any) => e.id)
+    await entity.delete(ids)
+  }
+}
+
+const run = async () => {
+  const server = await createServer(context)
+  // const seedClient = createTestClient(server.apollo)
+  const { con } = server
```

Comment:
Why are you creating a complete apollo server here if you only need the database connection? 

```suggestion
  // open mysql connection
  const con = await Connection.getInstance()
  if (!con?.isConnected) {
    logger.fatal(`Couldn't open connection to database!`)
    throw new Error(`Fatal: Couldn't open connection to database`)
  }

  // check for correct database version
  const dbVersion = await checkDBVersion(CONFIG.DB_VERSION)
  if (!dbVersion) {
    logger.fatal('Fatal: Database Version incorrect')
    throw new Error('Fatal: Database Version incorrect')
  }
```